src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java

```diff
@@ -77,21 +77,27 @@ public static void onEntityUpdate(LivingEvent.LivingUpdateEvent event) {
                 }
             }
         }
-//        if (eventEntityLiving.isPotionActive(ModPotions.boost))
-//        {
-//            int i = eventEntityLiving.getActivePotionEffect(ModPotions.boost).getAmplifier();
-//            {
-//                float percentIncrease = (i + 1) * 0.05f;
-//
-//                if (eventEntityLiving instanceof EntityPlayer)
-//                {
-//                    EntityPlayer entityPlayer = (EntityPlayer) eventEntityLiving;
-//
-//                    if ((entityPlayer.onGround || entityPlayer.capabilities.isFlying) && entityPlayer.moveForward > 0F)
-//                        entityPlayer.moveFlying(0F, 1F, entityPlayer.capabilities.isFlying ? (percentIncrease / 2.0f) : percentIncrease);
-//                }
-//            }
-//        }
+
+        if (event.getEntityLiving().isPotionActive(RegistrarBloodMagic.BOOST)) {
+            float percentIncrease = 0;
+            int i = event.getEntityLiving().getActivePotionEffect(RegistrarBloodMagic.BOOST).getAmplifier();
+            {
+                percentIncrease += (i + 1) * 0.5F;
+            }
```

Comment:
Minor suggestions:
- The float could just be declared on line 85 since there is only a single calculation.
- The int on line 83 could use a better name than I, will make future reference easier to see what the value is just referring to the name.